apiserver/appcat/update.go

```diff
@@ -0,0 +1,15 @@
+package appcat
+
+import (
+	"context"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apiserver/pkg/registry/rest"
+)
+
+var _ rest.Updater = &appcatStorage{}
+var _ rest.CreaterUpdater = &appcatStorage{}
+
+func (s appcatStorage) Update(ctx context.Context, name string, objInfo rest.UpdatedObjectInfo, createValidation rest.ValidateObjectFunc, updateValidation rest.ValidateObjectUpdateFunc, forceAllowCreate bool, options *metav1.UpdateOptions) (runtime.Object, bool, error) {
+	return nil, false, nil
```

Comment:
Nitpick: This will result in a weird error when someone tries to update an appcat listing.

Reply:
Or return an error that says it's not implemented?

Reply:
Do we plan to make AppCat objects updatable? What we expose via AppCat are composition annotations probably should not be updated.

Reply:
Not sure how this question relates to the discussion...

But returning an error that says that it's not implemented doesn't imply that there will be an implementation.